subsys/bluetooth/host/hci_core.c

```diff
@@ -3388,6 +3388,13 @@ static void hci_cmd_status(struct net_buf *buf)
 	}
 }
 
+static void hci_data_buf_overflow(struct net_buf *buf)
+{
+	struct bt_hci_evt_data_buf_overflow *evt = (void *)buf->data;
+
+	BT_DBG("link type %u", evt->link_type);
```

Comment:
I know I said BT_DBG in my previous comment, but I suppose getting implies that something is not quite right, so perhaps BT_WARN() is better?

Reply:
OK, that will be Monday